x-pack/plugins/security_solution/public/management/components/console/console.tsx

```diff
@@ -84,10 +84,12 @@ const ConsoleWindow = styled.div`
     &.euiDescriptionList {
       > .euiDescriptionList__title {
         width: 20%;
+        margin-top: ${({ theme: { eui } }) => eui.euiSizeS};
       }
 
       > .euiDescriptionList__description {
         width: 80%;
+        margin-top: ${({ theme: { eui } }) => eui.euiSizeS};
```

Comment:
You can add the `margin-top` property just once to the top level `&.euiDescriptionList` class instead of duplicating here.

Reply:
@ashokaditya I want the style on the individual row items, not the containing `euiDescriptionList`

Reply:
Oh, I see! Then it's good 👍